.github/workflows/publish-to-website.yml

```diff
@@ -0,0 +1,56 @@
+name: Publish to website
+
+env:
+  BRANCH_NAME: publish-latest-learning-path
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/publish-to-website.yml
+      - config/urls.yaml
+      - contributor/**
+      - introduction/**
+      - product-owner/**
+      - scripts/**
+      - trusted-committer/**
+      - workbook/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Learning Path
+        uses: actions/checkout@v2
+        with:
+          path: ./InnerSourceLearningPath
+      - name: Checkout innersourcecommons.org
+        uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.WEBSITE_DEPLOY_KEY }}
+          path: ./innersourcecommons.org
+          repository: InnerSourceCommons/innersourcecommons.org
+          fetch-depth: 0
+      - name: Branch innersourcecommons.org
+        working-directory: innersourcecommons.org
+        run: |
+            git checkout --track origin/$BRANCH_NAME || git checkout -b $BRANCH_NAME
+            git merge -X theirs origin/master --no-edit
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Build
+        working-directory: InnerSourceLearningPath/scripts
+        run: |
+            npm ci
+            TOKEN=${{ secrets.GITHUB_TOKEN }} node generate_learning_path_markdown.js
+            cp -r newsite/. ../../innersourcecommons.org/content/learn/learning-path/
+      - name : Commit
```

Comment:
Is there a way to open a PR or do we still have to do that manually?

Reply:
Maybe we should just list directly to _main_ and skip the whole branch thing.

Reply:
Discussed in https://github.com/InnerSourceCommons/innersourcecommons.org/pull/130, but to summarise:
- Deploy Keys do not have GitHub API access, so this Action cannot raise a PR. Instead, there is another Action on the innersourcecommons.org repo which raises a PR whenever the `publish-latest-learning-path` branch is pushed to (added in https://github.com/InnerSourceCommons/innersourcecommons.org/pull/130). We considered alternatives to using a Deploy Key, but stuck with this solution.
- Original plan was to push directly to main, but this would trigger a website rebuild into production and the preference was to have a manual step to reduce the risk of an outage of the website.